### PR TITLE
Feature zs 302 upgrade openiec61850 lib to1.5.0

### PR DESCRIPTION
**Pull request by bjornamkreutz**
ZS-302

### CODE REVIEW

**Comment by bjornamkreutz** [comment]
retest this please

File: cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceBattery.feature [hunk 1]
```diff
@@ -55,34 +55,24 @@ Feature: Microgrids Receive reports for Battery
...
-      | MeasurementNode_1_15   | Wrn4                     |
+      | MeasurementNode_1_1    | <Measurement_Node>       |
+
+    Examples: 
+      | Measurement_Node | Report_Trigger_Node |
+      | Beh              | LLN0.Beh.q          |
+      | Health           | LLN0.Health.q       |
+      | GnOpSt           | DGEN1.GnOpSt.q      |
+      | OpTmsRs          | DGEN1.OpTmsRs.q     |
+      #     | OutWSet    | DRCC1.OutWSet.subQ |
```
Comment (robindenadel) [inline cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceBattery.feature]: Create a new story to figure out how to trigger a report on a subQ value. Add some comments here to explain why this line is not active.

File: cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceBoiler.feature [hunk 2]
```diff
@@ -63,42 +59,24 @@ Feature: Microgrids Receive reports for Boiler
...
-      | MeasurementNode_1_18   | SchdId                   |
-      | MeasurementId_1_19     |                        4 |
-      | MeasurementNode_1_19   | SchdId                   |
+      | MeasurementNode_1_1    | <Measurement_Node>       |
+
+    Examples: 
+      | Measurement_Node | Report_Trigger_Node |
+      | Beh              | LLN0.Beh.q          |
+      | Health           | LLN0.Health.q       |
+      #     | OutWSet     | DRCC1.OutWSet.subQ |
```
Comment (robindenadel) [inline cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceBoiler.feature]: Add some comments here to explain why this line is not active.

File: cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceHeatPump.feature [hunk 3]
```diff
@@ -65,42 +60,24 @@ Feature: Microgrids Receive reports for Heat Pump
...
-      | MeasurementId_1_19     |                        4 |
-      | MeasurementNode_1_19   | SchdId                   |
+      | MeasurementNode_1_1    | <Measurement_Node>       |
+
+    Examples: 
+      | Measurement_Node | Report_Trigger_Node |
+      | Beh              | LLN0.Beh.q          |
+      | Health           | LLN0.Health.q       |
+      | GnOpSt           | DGEN1.GnOpSt.q      |
+      #     | OutWSet     | DRCC1.OutWSet.subQ |
```
Comment (robindenadel) [inline cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceHeatPump.feature]: Add some comments here to explain why this line is not active.

File: cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServicePq.feature [hunk 4]
```diff
@@ -23,94 +23,148 @@ Feature: Microgrids Receive reports for PQ
...
+      | IntIn2           | GGIO1.IntIn2.q      |
+      | Alm1             | GGIO1.Alm1.q        |
+      | Alm2             | GGIO1.Alm2.q        |
+      | Alm3             | GGIO1.Alm3.q        |
+      | Alm4             | GGIO1.Alm4.q        |
+      | Wrn1             | GGIO1.Wrn1.q        |
+      | Wrn2             | GGIO1.Wrn2.q        |
+      | Wrn3             | GGIO1.Wrn3.q        |
+      | Wrn4             | GGIO1.Wrn4.q        |
+      #| OpCntRs     | QVVR1.OpCntRs.ctlModel   |
```
Comment (robindenadel) [inline cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServicePq.feature]: Add some comments here to explain why this line is not active.

File: cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServicePv.feature [hunk 5]
```diff
@@ -53,34 +54,24 @@ Feature: Microgrids Receive reports for PV
...
-      | MeasurementNode_1_14   | Wrn3                     |
-      | MeasurementId_1_15     |                        1 |
-      | MeasurementNode_1_15   | Wrn4                     |
+      | MeasurementNode_1_1    | <Measurement_Node>       |
+
+    Examples: 
+      | Measurement_Node | Report_Trigger_Node |
+      | Beh              | LLN0.Beh.q          |
+      | Health           | LLN0.Health.q       |
+      #     | OutWSet     |DRCC1.OutWSet.subQ |
```
Comment (robindenadel) [inline cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServicePv.feature]: Add some comments here to explain why this line is not active.

File: cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceWind.feature [hunk 6]
```diff
@@ -71,34 +88,24 @@ Feature: Microgrids Receive reports for Wind
...
-      | MeasurementNode_1_14   | Wrn3                     |
-      | MeasurementId_1_15     |                        1 |
-      | MeasurementNode_1_15   | Wrn4                     |
+      | MeasurementNode_1_1    | <Measurement_Node>       |
+
+    Examples: 
+      | Measurement_Node | Report_Trigger_Node |
+      | Beh              | LLN0.Beh.q          |
+      | Health           | LLN0.Health.q       |
+      #     | OutWSet     | DRCC1.OutWSet.subQ |
```
Comment (robindenadel) [inline cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/reporting/PushServiceWind.feature]: Add some comments here to explain why this line is not active.

**Pull request merged**